beanstalk_worker/worker_start_build.py

```diff
@@ -31,88 +31,95 @@
 DEBUG=1
 
 def debug_log(msg):
-  if DEBUG==1:
-    logger.log(level=logging.INFO,msg=msg)
+    if DEBUG==1:
+        logger.log(level=logging.INFO,msg=msg)
```

Comment:
You use ``logger.info`` for info logs and ``logger.debug`` for debug logs. You can set the log level during run time or from config.